code-challenges.test.js

```diff
@@ -65,5 +147,43 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+describe("vowelIndex", () => {
+  it("takes in a string and logs the index of the first vowel", () => {
+    expect(vowelIndex(vowelTester1)).toEqual(1)
+    expect(vowelIndex(vowelTester2)).toEqual(0)
+    expect(vowelIndex(vowelTester3)).toEqual(2)
+  })
+})
+
+// ● vowelIndex › takes in a string and logs the index of the first vowel
+//
+//   ReferenceError: wordsCapitilized is not defined
 
 // b) Create the function that makes the test pass.
+
+//Create a new array with all vowels
+//Create a function - vowelIndex
+//Parameter- string
+//Create new variable to store the new array data
+//Create a for loop to run through the strings given
+//Create a conditional to identify the vowels in the strings given
+//Utilize the .push() method to push the new data in the variable created
+//Return the vowels, use the .sort() method to have the index of the vowels in value order. Add an index[0] to only return the first index.
+//Create code to take in all variables given above
+
+const vowel = ["a", "e", "i", "o", "u"]
+
+const vowelIndex = (string) => {
+  let vowelLog = []
+  for (let i = 0; i < string.length; i++) {
+    if (vowel.indexOf(string[i]) > -1) {
+      vowelLog.push(i);
+      }
+    }return vowelLog.sort()[0]
+  }
```

Comment:
This is a pretty cool approach!  Since you are iterating using a for loop that goes in order and pushing the index into the array, you shouldn't need to use .sort at the end.  Nicely done!